src/Microsoft.AspNetCore.Mvc.Abstractions/Filters/IFilterProvider.cs

```diff
@@ -26,8 +29,16 @@ public interface IFilterProvider
         /// </remarks>
         int Order { get; }
```

Comment:
Ima let you finish. But these remarks are some of the best remarks of all time :sunglasses: